Modules/CLI/FiberTractMeasurements/FiberTractMeasurements.cxx

```diff
@@ -179,6 +179,12 @@ void computeScalarMeasurements(vtkSmartPointer<vtkPolyData> poly,
         continue;
         }
 
+      if (operation.find(std::string("FractionalAnisotropy")) != std::string::npos && val >= 1)
```

Comment:
yes. though in practice I don’t think we should ever be seeing an FA of 1.

but don’t accept this pull request yet. Only the output of the correct name for the scalar measures seems safe to me.

The thing that really needs to be looked into is how we could ever get a number greater than 1.

On Mar 28, 2016, at 6:07 PM, Isaiah <notifications@github.com<mailto:notifications@github.com>> wrote:


In Modules/CLI/FiberTractMeasurements/FiberTractMeasurements.cxx<https://github.com/SlicerDMRI/SlicerDMRI/pull/38#discussion_r57642067>:

> @@ -179,6 +179,12 @@ void computeScalarMeasurements(vtkSmartPointer<vtkPolyData> poly,
>          continue;
>          }
>
> +      if (operation.find(std::string("FractionalAnisotropy")) != std::string::npos && val >= 1)


Should this be > 1 instead of >= 1?

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/SlicerDMRI/SlicerDMRI/pull/38/files/9ff381e684c1a6d544459a36a5f44a594e4dc597#r57642067>



The information in this e-mail is intended only for the person to whom it is
addressed. If you believe this e-mail was sent to you in error and the e-mail
contains patient information, please contact the Partners Compliance HelpLine at
http://www.partners.org/complianceline . If the e-mail was sent to you in error
but does not contain patient information, please contact the sender and properly
dispose of the e-mail.